pyqg/layered_model.py

```diff
@@ -282,49 +282,49 @@ def _initialize_model_diagnostics(self):
         self.add_diagnostic('entspec',
                 description='barotropic enstrophy spectrum',
                 function= (lambda self:
-                    np.abs((self.Hi[:,np.newaxis,np.newaxis]*self.qh).sum(axis=0))**2/self.H),
-                units='',
+                    np.abs((self.Hi[:,np.newaxis,np.newaxis]*self.qh).sum(axis=0))**2/self.H/self.M**2),
+                units='meters second ^-2',
                 dims=('l','k')
         )
 
         self.add_diagnostic('KEspec_modal',
                 description='modal kinetic energy spectra',
                 function= (lambda self:
                     self.wv2*(np.abs(self.phn)**2)/self.M**2 ),
-                units='',
+                units='meters squared second ^-2',
```

Comment:
To be honest, I have always felt that our usage of units was a little to verbose. I would prefer to just see.
```suggestion
                units='m^2 s^-2',
```

(which is also pint and udunits compatible)

Would you mind doing a global search and replace to fix this everywhere?

Reply:
Sure, that sounds good to me! I was just mimicking what was already there.

Reply:
Updated!